src/Data/Array/Polarized/Push.hs

```diff
@@ -19,6 +20,16 @@ import Prelude.Linear
 import qualified Prelude
 import Data.Monoid.Linear
 
+-- XXX: the below isn't really true yet since no friendly way of constructing a
+-- PushArray directly is given yet (see issue #62), but the spirit holds.
+-- XXX: There's also a slight lie in that one might one to consume a PushArray
```

Comment:
might want

Reply:
Agreed, I'll rewrite these to be TODOs - it's not yet clear what a good way of doing this is.

Reply:
I meant "might one" -> "might want"

Reply:
Ah. My mistake, thanks for this catch.